.travis.yml

```diff
@@ -1,35 +1,14 @@
 language: python
+dist: xenial
```

Comment:
This MUST be done even if form some reason I do not understand this PR is refused!

Reply:
It's merely extremely highly recommended.